app/models/manageiq/providers/openshift/infra_manager/refresh_worker.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::Openshift::InfraManager::RefreshWorker < ManageIQ::Providers::Kubevirt::InfraManager::RefreshWorker
```

Comment:
We don't need to inherit anything from the Kubevirt RefreshWorker, and doing this nests the settings under kubevirt so we typically don't do this for simple workers (e.g. see Openshift::ContainerManager::RefreshWorker it doesn't < Kubernetes)
```suggestion
class ManageIQ::Providers::Openshift::InfraManager::RefreshWorker < ManageIQ::Providers::BaseManager::RefreshWorker
```

Reply:
You also need to add a settings section for this new worker to config/settings.yml
```
diff --git a/config/settings.yml b/config/settings.yml
index dc097791..448ed39b 100644
--- a/config/settings.yml
+++ b/config/settings.yml
@@ -33,3 +33,4 @@
           :ems_metrics_collector_worker_openshift: {}
         :ems_refresh_worker:
           :ems_refresh_worker_openshift: {}
+          :ems_refresh_worker_openshift_infra: {}
```

Reply:
NOTE we'll also need to add .service and .target files for the new openshift-infra-refresh worker in the systemd/ directory here (specs will catch this but just want to head it off now)